apps/system/lockscreen/js/lockscreen.js

```diff
@@ -708,6 +714,19 @@
   };
 
   /**
+   *  Sets UnlockSoundEnabled to true or false
+   *  when the screen gets unlocked
+   */
+  LockScreen.prototype.setUnlockSoundEnabled =
+  function lockscreen_setUnlockSoundEnabled(isUnlockSoundEnabled) {
+    if(isUnlockSoundEnabled) {
+      this.unlockSoundEnabled = true;
```

Comment:
With ```this.unlockSoundEnabled = !!(isUnlockSoundEnabled);``` we should not need a method anymore. Let me make those changes and test them.

Thanks.